projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-headline/order-detail-headline.component.html

```diff
@@ -23,5 +23,22 @@
         }}
       </div>
     </div>
+    <div
+      class="cx-detail col-sm-12 col-md-3"
+      *cxFeature="'cancellationAndReturn'"
+    >
+      <button
+        *ngIf="order.returnable"
+        [routerLink]="
+          {
+            cxRoute: 'orderReturn',
+            params: order
+          } | cxUrl
+        "
+        class="btn btn-action"
```

Comment:
From looking at the UX the return button has a solid color (primary or secondary) and cancel has a white background